packages/test/stochastic-test-utils/src/markovChain.ts

```diff
@@ -117,11 +113,8 @@ export class SpaceEfficientWordMarkovChain extends MarkovChain<string, string> {
 					}
 
 					const currentWordCount = initialChain[prevWord][word];
-					if (currentWordCount !== undefined) {
-						initialChain[prevWord][word] = currentWordCount + 1;
-					} else {
-						initialChain[prevWord][word] = 1;
-					}
+					initialChain[prevWord][word] =
+						currentWordCount !== undefined ? currentWordCount + 1 : 1;
```

Comment:
```suggestion
						currentWordCount === undefined ? 1: currentWordCount + 1;
```

Reply:
Maybe the no negated condition rule doesn't apply to ternary expressions.